Find a functionally equivalent code:android.view.accessibility.AccessibilityNodeInfo.getChildCount:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the number of children.
 *
 * @return The child count.
 */

@@ -1,3 +1,3 @@
 {
-    return mChildNodeIds.size();
+    return mChildNodeIds == null ? 0 : mChildNodeIds.size();
 }

